plugins/node/plugin.zsh

```diff
@@ -0,0 +1,18 @@
+# Color definitions
+GEOMETRY_COLOR_NODE_NPM_VERISON=${GEOMETRY_COLOR_NODE_NPM_VERISON:-black}
+
+# Symbol definitions
+GEOMETRY_SYMBOL_NODE_NPM_VERISON=${GEOMETRY_SYMBOL_NODE_NPM_VERSION:-"⬡"}
+GEOMETRY_NODE_NPM_VERISON=$(prompt_geometry_colorize $GEOMETRY_COLOR_NODE_NPM_VERISON GEOMETRY_SYMBOL_NODE_NPM_VERISON) 
+
+geometry_prompt_node_setup() 
+
+geometry_prompt_node_render(){
+	local node_path="$echo "$(which node)" -v"
+	local npm_path="$echo "$(which npm)" --version"
+	GEOMETRY_NODE_VERSION=`(eval ${node_path})`
```

Comment:
`$+commands[node]` will return `1` if it finds `node` in the `commands` array. Example:

With `node` command available in `$PATH`:

    > echo $+commands[node]
    1

Without `node` available:

    > echo $+commands[node]
    0

If node is found in `$+commands` we should be able to do `node -v`, otherwise I'd return immediately without displaying anything.